### PR TITLE
control point type propagation on to_isis

### PR DESCRIPTION
**Pull request by Kelvinrr**
Goes with an update to plio

### CODE REVIEW

File: autocnet/matcher/subpixel.py [hunk 1]
```diff
@@ -237,42 +237,56 @@ def iterative_phase(sx, sy, dx, dy, s_img, d_img, size=251, reduction=11, conver
...
+        # Apply the shift to d_search and compute the new correspondence location
+        dx += (shift_x + dxr)
+        dy += (shift_y + dyr)
+
+        # Break if the solution has converged
+        size -= reduction
+        dist = np.linalg.norm([dsample-dx, dline-dy])
+        if size <1:
+            return None, None, None
+        if abs(shift_x) <= convergence_threshold and\
```
Comment (Kelvinrr) [inline autocnet/matcher/subpixel.py]: seems like the only way to simulate do while in python?

File: autocnet/matcher/cross_instrument_matcher.py [hunk 2]
```diff
@@ -168,21 +168,27 @@ def themis_ground_to_ctx_matcher(cnet):
                     match_results.append(ex)
                     continue
 
-                if ret is not None:
+                if ret is not None and None not in ret:
```
Comment (jlaura) [inline autocnet/matcher/cross_instrument_matcher.py]: This is legit. If self is not the soul and soul is not in the self.

File: autocnet/matcher/cross_instrument_matcher.py [hunk 3]
```diff
@@ -168,21 +168,27 @@ def themis_ground_to_ctx_matcher(cnet):
...
                     x,y,metrics = ret
                 else:
                     match_results.append("Failed to Converge")
                     continue
-
-                dist = np.linalg.norm([x-dx, -1*(y-dy)])
-                match_results.append([k, x-dx, -1*(y-dy), dist])
+                try:
+                  dist = np.linalg.norm([x-dx, -1*(y-dy)])
```
Comment (jlaura) [inline autocnet/matcher/cross_instrument_matcher.py]: Under what case is the try necessary?

File: autocnet/matcher/subpixel.py (hunk 1 above)
Comment (jlaura) [reply autocnet/matcher/subpixel.py]: The alternative would be to do this in the `while` above and prepopulating the vals with infs. Having said that, this looks fine to me too.

**Comment by jlaura** [comment]
One comments. Let's merge this and plio concurrently.

File: autocnet/matcher/cross_instrument_matcher.py (hunk 3 above)
Comment (Kelvinrr) [reply autocnet/matcher/cross_instrument_matcher.py]: I think it was from the two return None originally. Should be removed.

**Pull request merged**